src/snakemake/workflow.py

```
@@ -19,7 +19,7 @@
 import tarfile
 import tempfile
 from typing import Dict, Iterable, List, Optional, Set, Union
-from snakemake.io.access_patterns import access_pattern
+from snakemake.io.access_patterns import AccessPatternFactory, access_pattern
```

Ruff (0.8.2)

`snakemake.io.access_patterns.access_pattern` imported but unused

Remove unused import: `snakemake.io.access_patterns.access_pattern`

(F401)